rules/windows/process_creation/win_susp_splwow64.yml

```diff
@@ -0,0 +1,20 @@
+title: Suspicious Splwow64 Without Params
+id: 1f1a8509-2cbb-44f5-8751-8e1571518ce2
+status: experimental
+description: Detects suspicious Splwow64.exe process without any command line parameters
+references:
+    - https://twitter.com/sbousseaden/status/1429401053229891590?s=12
+author: Florian Roth
+date: 2021/08/23
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        Image|endswith: '\splwow64.exe'
+    filter:
+        CommandLine|contains: 'splwow64.exe '
```

Comment:
while not `CommandLine|endswith: 'splwow64.exe'`  to use with no args (the args.count <2)

Reply:
Yes, better, thanks

Reply:
Hi, 

I dont think so, because the Image never contains parameters just image name (the exe), the CommandLine will be the rigth field...